wof/examples/production_configs/wofpy.conf

```diff
@@ -15,5 +15,5 @@ setgid www-data
 env PATH=/home/ubuntu/miniconda/envs/wofpy/bin
 
 # change directory to where the uWSGI configuration file is located then execute uwsgi with the config file
-chdir /var/www/timeseries
-exec uwsgi --ini timeseries.ini
\ No newline at end of file
+chdir /var/www/wofpy
+exec uwsgi --ini wofpy.ini
```

Comment:
> The actual implementation of multiple dao's would be in the runserver script. In this case you'd use the runserver_multiple.py

I missed this when I made my comments ... 

It looks like the file `wofpy` would also need to be modified for a multi-service set up, no?